feat: track http request data in http trigger (#448)
arcus-azure/arcus.observability: docs/preview/03-Features/writing-different-telemetry-types.md

> 💡 Note that [Arcus Web API request tracking middleware](https://webapi.arcus-azure.net/features/logging#logging-incoming-requests) can already do this for you in a ASP.NET Core application

> ### Incoming HTTP requests in Azure Function HTTP trigger

Reviewer: ```suggestion
### Incoming HTTP requests in HTTP triggered Azure Functions
```

Author: Is that better? 'triggered functions' doesn't seem like an improvement here?

Reviewer: Well, I was thinking:  the Azure Function is receiving the http request (it is triggered by an http trigger).  It is not the HTTP Trigger that receives the request.

Author: It's maybe more to read like Service Bus topic, for example.
Microsoft uses this in their title: https://docs.microsoft.com/en-us/azure/azure-functions/functions-bindings-http-webhook-trigger?tabs=in-process%2Cfunctionsv2&pivots=programming-language-csharp

Reviewer: That's really documentation on the Http Trigger (binding) of a Function.  So, imho, that is something different.
But, for me it is OK.  I just thought it was clearer / more correct.

Author: When we complete our service correlation docs we could maybe agree on a common naming convention we use for pur components and update the docs in the different libraries accordingly. 👍

Requests allow you to keep track of the HTTP requests that are performed against your Azure Function HTTP trigger and what the response was that was sent out.

**Installation**

> If you want to track the `HttpRequestData` of an isolated Azure Functions HTTP trigger project, you'll have to install an additional package to include these dependencies:

Reviewer: ```suggestion
For isolated Azure Functions, you'll have to install an additional package if you want to track the `HttpRequestData`:
```
